react-app/src/Finish/Buttons.js

```diff
@@ -0,0 +1,84 @@
+import { Alert, Pressable, StyleSheet, Text, View } from 'react-native'
+import { useNavigation } from '@react-navigation/native'
+import React from 'react'
+import ArrowRotateLeft from '../../assets/icons/arrow-rotate-left.svg'
+import Home from '../../assets/icons/home.svg'
+import Train from '../../assets/icons/train.svg'
+
+export default function Buttons() {
```

Comment:
Three buttons share same looks and structure. We can consider abstracting to reduce duplication and enable reuse in future

Reply:
Yep I also think so, but the tricky thing is to handle `svg` file inside of the components. Let me dig a little deeper.